list.rb

```diff
@@ -33,9 +37,24 @@ def toggle_item(name)
   def add(name)
     self.items << Item.new(name)
   end
-
+  
   def save!
     lines = [name] + @items.map(&:display_line)
     File.write(filename, lines.join("\n"))
   end
+  def self.load_all_lists
+    lists = []
+    files = Dir["./data/*.md"]
+    files.map do |file|
+      list = new("#{file[7]}")
+      list.load_from_file
+      lists << list  
+    end
+    lists
```

Comment:
this is ok but using `file[7]` means you can only support 0.md, 1.md to 9.md. Alternatively you can get the id by splitting the file name by `"/"`, then pick the last part of the string

```ruby
# "data/0.md" => 0
# "data/10.md" => 10
file.split("/").last.to_i
```